dispatcher/dispatcher.go

```diff
@@ -56,6 +64,12 @@ func (cfg Config) Start(q JobQueue, m *model.Queue) Dispatcher {
 	}
 	w := wc.NewWorker()
 
+	dps := rate.Limit(m.MaxDispatchesPerSecond)
+	if dps == 0 {
+		dps = rate.Inf
+	}
+	limiter := rate.NewLimiter(dps, int(m.MaxBurstSize))
```

Comment:
When MaxDispatchesPerSecond is empty, we set it to Inf, meaning no limit. Though MaxBurstSize is also empty in such case, we don't have to care it because it is ignored.

https://pkg.go.dev/golang.org/x/time/rate#pkg-constants
> Inf is the infinite rate limit; it allows all events (even if burst is zero).